db/migrate/20211109200705_backfill_type_on_vita_partner.rb

```diff
@@ -0,0 +1,15 @@
+class BackfillTypeOnVitaPartner < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
```

Comment:
I encourage you to keep transactions enabled. `disable_ddl_transaction!` is required for `algorithm: :concurrently`, but you're not doing that, so let the transactions flow.

Not a blocker... but would you be willing to change it?